gapir/cc/memory_manager.cpp

```diff
@@ -32,13 +32,12 @@
 //   ┃             ┃         volatile       ┃
 //   ┃   mMemory   ┃          memory        ┃
 //   ┃             ┃                        ┃
-//   ┃             ┣━━━━━━━━━━━━┳━━━━━━━━━━━┨
-//   ┃             ┃            ┃  constant ┃
-//   ┃             ┃   replay   ┃   memory  ┃
-//   ┃             ┃    data    ┣━━━━━━━━━━━┨
-//   ┃             ┃            ┃   opcode  ┃
-//   ┃             ┃            ┃   memory  ┃
-//   ┗━━━━━━━━━━━━━┻━━━━━━━━━━━━┻━━━━━━━━━━━┛
+//   ┃             ┃                        ┃
```

Comment:
Need to update this graph, as we swapped the position of these two segments.

Reply:
Done